### ISSUE TITLE
bug(naijatype): supported language description for Naija Type keyboard using the `<pre` tag results in unresponsive styling

### ISSUE THREAD

**Issue by Meng-Heng** [keymanapp/keyboards#3021]
![image](https://github.com/user-attachments/assets/7c2233b6-434c-40b4-b8ed-6d0cd4ca5c4e)
![image](https://github.com/user-attachments/assets/b111bf5a-305e-4de3-984f-5490e67bc465)
I only stumbled one keyboard so far.

**Comment by mcdurdin** [comment]
https://keyman.com/keyboards/naijatype

This can be addressed by removing the start-of-line whitespace in the `<Description>` element:

https://github.com/keymanapp/keyboards/blob/618fe56c8e5e81f1767c030f6a7a0aa80b142cdc/experimental/n/naijatype/source/naijatype.kps#L27-L31

Something like:

```xml
    <Description URL="">This keyboard helps you to type any letters used in 100+
Nigerian orthographies including under-dots, underlines, diacritics, tone
marks, different shaped letters like a̱ ẹ ɨ ʉ ʊ́ ɓ ɦ m̃ ŋ ₦ and many others
all in one keyboard for computers or touch.

Languages covered include: Abua, Adara, Agatu, Agwagwune, Alago, Ashe, Atsam,
Awak, Bacama, Bali, Bangwinji, Basa, Bekwarra, Berom, Bokobaru, Bura-Pabir,
Burak, Cahungwarya, Cakfem-Mushere, Cishingini, C’Lela, Dadiya, Dghwede, Dikaka,
Duya, Dza, Ebira, Edo, Ekajuk, Engenni, English, Etkywan, Etulo, Ezaa, Nigerian
Fulfulde, Gbari, Goemai, Gokana, Gude, Gwak, Hausa, Hyam, Ibaas, Ibani, Idoma,
Igbo, Igede, Ika, Ikwere, Ikwo, Iten, Ivbie North-Okpela-Arhe, Izere, Izii,
Jibu, Jju, Kamuku, Kamwe, Yerwa Kanuri, Karekare, Kibaku, Kirike, Kukele, Kutep,
Lamang, Lokaa, Longuda, Mada, Nigeria Mambila, Mbe, Cross River Mbembe,
Mbula-Bwazza, Mumuye, Mwaghavul, Ngas, Ninzo, Nupe-Nupe-Tako, Nya Huba, Obolo,
Odual, Ogbah, Pangu, Pero, Nigerian Pidgin, Pyam, Reshe, Rigwe, Ron, Tal,
Tangale, Tarok, Tera, Tiv, Tsikimba, Tsishingini, Tsuvadi, Tula, Tyap, ut-Hun,
ut-Ma’in, Waja, Warji, Yala, Yoruba

This is the package for the keyboard. Lexical models (text completion) for
Hausa, English, Ishɛ, Pangu, Fulfulde etc are packaged separately.</Description>
```

**Comment by mcdurdin** [comment]
Side note: this might be a good lint rule for the package compiler -- prevent use of indenting for code blocks in Markdown found in the `<Description>` element?

**Comment by Meng-Heng** [comment]
You know best on this, Marc. Now, should I edit the kps file for this or would your implementation correct the indent for the keyboard?

**Comment by mcdurdin** [comment]
> Now, should I edit the kps file for this or would your implementation correct the indent for the keyboard?

Yes, go ahead and edit the .kps file. We won't fix this in Keyman Developer until 18.0 release, so best to patch the issue in the .kps first. Don't forget to bump the version number of the keyboard.
